app/com/arpnetworking/rollups/RollupGenerator.java

```diff
@@ -301,6 +307,17 @@ private void handleLastDataPointsMessage(final LastDataPointsMessage message) {
                     period
             );
 
+            if (!startTimes.isEmpty()) {
```

Comment:
**OR** Why isn't this periodic metrics?

Reply:
It wasn't periodic metrics because the tag needs to be different per sample observed, since it's extracted from the metric name.

Reply:
👍